config/application.yml.default

```diff
@@ -428,6 +429,7 @@ development:
   state_tracking_enabled: true
   telephony_adapter: test
   use_dashboard_service_providers: true
+  use_reauthentication_route: true
```

Comment:
Since the default is `true`, it's not strictly necessary to override here in `development` and in the `test` block below.

```suggestion
```